verl/trainer/ppo/ray_trainer.py

```diff
@@ -1070,13 +1097,17 @@ async def fit(self):
                         # compute rewards. apply_kl_penalty if available
                         if self.config.algorithm.use_kl_in_reward:
                             batch, kl_metrics = apply_kl_penalty(
-                                batch, kl_ctrl=self.kl_ctrl_in_reward, kl_penalty=self.config.algorithm.kl_penalty
+                                batch,
+                                kl_ctrl=self.kl_ctrl_in_reward,
+                                kl_penalty=self.config.algorithm.kl_penalty,
+                                multi_turn=self.config.actor_rollout_ref.rollout.multi_turn.enable,
```

Comment:
Remove multi_turn key as single turn is just a special case of multiturn

Reply:
A separate pull request will be created to convert the response_mask used as loss_mask into the loss_mask key.